### DISCUSSION TITLE
Couldn't find AnalyzedNuxtPageMeta by NuxtPage

### DISCUSSION THREAD

**Discussion by memic84** [nuxt/nuxt#26801] [Questions]
I get a lot of these lately during dev

WARN [@nuxtjs/i18n]: Couldn't find AnalyzedNuxtPageMeta by NuxtPage (orders), so no custom route for it

I am using the extends feature of Nuxt, and the pages which give a warning are those which exist in the core folder, but not in the app (the one from core is loaded there). It seems to me that, Nuxt or Nuxt-i18n doesn't take this into account, or does the check before copied the files.

Does anybody know if this is a Nuxt bug or i18n package bug?

**Comment by leopoldkristjansson** [comment]
Just to chip in, I am using Prismic as a CMS and I get the following Warning for my /preview route.

`[12:28:33]  WARN  [@nuxtjs/i18n]: Couldn't find AnalyzedNuxtPageMeta by NuxtPage (/preview), so no custom route for it`

This is a part of my nuxt.config.

```ts
import consts from './consts'

export default defineNuxtConfig({
  modules: [
    '@nuxtjs/prismic',
  ],
  i18n: {
    baseUrl: consts.baseUrl,
    locales: [
      {
        code: 'en-us',
        language: 'en',
      },
      {
        code: 'de-de',
        language: 'de',
      },
    ],
    customRoutes: 'config',
    detectBrowserLanguage: false,
    defaultLocale: 'en-us',
    vueI18n: './i18n.config.ts',
  },
  prismic: {
    endpoint: 'example-com',
    preview: '/preview',
  },
})

```

If anyone knows what needs to be (or can be) configured to get rid of the Warning, I would be happy to hear about it.

**Comment by mrc-bsllt** [comment]
Same, any update?
